# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 017a5fb444d4
# Full Hash: 017a5fb444d4f5e1eb1772cd89db2244afa8cce4
# Author: Cristina Horotan <chorotan@mozilla.com>
# Date: 2025-06-13 09:36:48
# Regressor Bug: 1945683
# File Overlap Count: 1
# Description:
#   Revert "Bug 1945683 - Enable layer compositor on Windows in Nightly r=gfx-reviewers,lsalzman" for causing reftest failures
#   
#   This reverts commit 5daef99508f341552c9b2441b46c50f5bec3a22b.
# ==============================================================================

diff -r 625988823050 -r 017a5fb444d4 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Fri Jun 13 07:03:22 2025 +0300
+++ b/modules/libpref/init/StaticPrefList.yaml	Fri Jun 13 08:57:44 2025 +0300
@@ -7530,14 +7530,14 @@
 #endif
   mirror: once
 
+#ifdef NIGHTLY_BUILD
+  # Keep this pref hidden on non-nightly builds to avoid people accidentally
+  # turning it on.
 - name: gfx.webrender.layer-compositor
   type: bool
-#if defined(XP_WIN)
-  value: @IS_NIGHTLY_BUILD@
-#else
-  value: false
-#endif
-  mirror: once
+  value: false
+  mirror: once
+#endif
 
 #ifdef XP_WIN
 - name: gfx.webrender.layer-compositor-use-composition-surface
